src/common/types/common.ts

```diff
@@ -0,0 +1,9 @@
+export enum LocatorType {
+  cssSelector = "CSS selector",
+  xPath = "xPath",
+}
```

Comment:
why do you move it to common? it's all about `locators` feature

Reply:
I believe that we can move it common, because it is used not only in locator feature